cddl/accepted-claims-set.cddl

```diff
@@ -0,0 +1,6 @@
+accepted-claims-set = {
+  &(evidence-triples: 0) => [ + reference-triple-record ]
+  ? &(identity-triples: 2) => [ + identity-triple-record ]
+  ? &(coswid-triples: 6) => [ + ev-coswid-triple-record ]
```

Comment:
There is no reason for the indexes to mirror CoMID's: this is a completely separate namespace.  I suggest we move to 0, 1, 2, ... instead.

Reply:
ok, done